File: Programmers/핸드폰 번호 가리기.cpp
#include <iostream>
#include <string>
using namespace std;

string solution(string phone_number) {
	for (int i = 0; i < phone_number.size() - 4; i++)
		phone_number[i] = '*';

	return phone_number;
}

int main() {
	string test("01055536684");
	cout << solution(test);
	system("pause");
}

File: Programmers/핸드폰 번호 가리기.java
class Solution {
    public String solution(String phone_number) {
        String answer = "";
        
        for(int i = 0 ; i < phone_number.length(); i++)
            if (i == phone_number.length() - 4) {
                answer += phone_number.substring(phone_number.length() - 4, phone_number.length());
                break;
                }
            else
                answer += '*';
                
        return answer;
    }
}